draft-demarco-oauth-nonce-endpoint.md

```diff
@@ -41,13 +45,19 @@ informative:
   BCP195: BCP195
 
 --- abstract
-This document defines the Nonce Endpoint for OAuth 2.0 implementations [RFC6749]. It details how an Authorization Server generates and issues opaque Nonces and how a client can learn about this endpoint to obtain a Nonce generated an Authorization Server or another entity.
+This document details how a client can learn about an endpoint to obtain a Nonce generated by a server (Nonce endpoint) and provides security guidelines for server-provided Nonce generation and verification.
```

Comment:
I am not in favor of using (brackets) in abstract text

Reply:
There are three main changes: 1. the removal of "OAuth 2.0 implementations", 2. the removal of the term "opaque", 3. the addition of the security guidelines. Regarding:

1. I was wondering if at least in the abstract we could generalize the contribution and specify only in the introduction that this endpoint can be used in OAuth 2.0 implementations 
2. I suggest to avoid introducing nonce properties here
3. I added this part as is one of the two contributions of this specification

Regarding the use of brackets there is no problem, we can rewrite it differently. As soon as we agree on the content, I will propose a new text.

Reply:
I suggest keeping OAuth out of the abstract and intro, but then putting it in a top level section before the rest of the content.

Reply:
I'm going to accept your suggestion here https://github.com/peppelinux/draft-demarco-oauth-nonce-endpoint/pull/18 and merge it

then this PR will be closed OR revamped after an aligned with the main branch